.github/workflows/feature.yml

```diff
@@ -0,0 +1,68 @@
+name: Feature Test
+on:
+  push:
+    branches:
+      - feature/**
+      - main
+jobs:
+  sfdx_scanner:
+    name: "Run SFDX Scanner PMD"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install sfdx
+        run: |
+          mkdir sfdx
+          wget -qO- https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz | tar xJ -C sfdx --strip-components 1
+          ./sfdx/install
+          sfdx plugins:install @salesforce/sfdx-scanner
+      - name: Run Analysis
+        run: |
+          sfdx scanner:run -t dlrs -e pmd -f junit -s 2 > results.xml
```

Comment:
Have you run this locally ? I will try as soon as time permits. Wonder how many things we need to ignore or clean up. 
As it stands, the test fail message for pmd is not very helpful:

```ERROR running scanner:run:  Detected rule violations of severity 2 or more severe.```

Reply:
You're right, this is not very useful. I will tweak the job to always store the artifact so we can reference it from a job.